ArchiSteamFarm/Actions.cs

```diff
@@ -244,9 +244,9 @@ public sealed class Actions : IAsyncDisposable {
 		}
 
 		[PublicAPI]
-		public async Task<(bool Success, string Message)> SendInventory(uint appID = Steam.Asset.SteamAppID, ulong contextID = Steam.Asset.SteamCommunityContextID, ulong targetSteamID = 0, string? tradeToken = null, Func<Steam.Asset, bool>? filterFunction = null) {
-			if ((appID == 0) || (contextID == 0)) {
-				throw new ArgumentNullException(nameof(appID) + " || " + nameof(contextID));
+		public async Task<(bool Success, string Message)> SendInventory(IReadOnlyCollection<Steam.Asset> items, ulong targetSteamID = 0, string? tradeToken = null) {
+			if (items.Count == 0) {
+				return (false, string.Format(Strings.ErrorIsEmpty, nameof(items)));
```

Comment:
You do not check `items` against `null`, and it be exception instead - if caller is brain-damaged to call this function with arguments that do not make sense, he's going to get yelled at.

Reply:
It's `IReadOnlyCollection<Steam.Asset>` and not `IReadOnlyCollection<Steam.Asset>?`, but I will throw an exception in case it is empty.

Reply:
Exactly, if it was `IReadOnlyCollection<Steam.Asset>?` you actually **couldn't** throw on null because it's expected use case. But you don't, you declare it as **not nullable**, therefore you should explicitly throw if it is - just because you announce it doesn't affect the compiler and userspace.